apps/test/unit/templates/sectionsRefresh/SectionsSetUpContainerTest.jsx

```diff
@@ -17,8 +17,8 @@ describe('SectionsSetUpContainer', () => {
     const wrapper = shallow(<SectionsSetUpContainer />);
 
     expect(wrapper.find('Heading1').length).to.equal(1);
-    expect(wrapper.find('Button').length).to.equal(3);
-    expect(wrapper.find('Button').at(2).props().text).to.equal(
+    expect(wrapper.find('Button').length).to.equal(4);
```

Comment:
Adding a button in the middle so all these indexes are wrong. Moving to .last so that we don't have to do this again.